app/questionnaire/questionnaire_schema.py

```diff
@@ -725,6 +753,26 @@ def get_first_answer_id_for_block(self, block_id: str) -> str:
         answer_ids = self.get_answer_ids_for_block(block_id)
         return answer_ids[0]
 
+    def get_answer_format_for_calculated_summary(
+        self, calculated_summary_block_id: str
+    ) -> dict:
+        """
+        Given a calculated summary block id, find the format of the total by using the first answer
+        """
+        answer_format = {"type": None}
+        if calculated_summary_block := self.get_block(calculated_summary_block_id):
```

Comment:
Is it possible for this to not return a block?. If not, remove the if and type ignore?

Reply:
Updated - it would only not return a block if the function was called with an invalid calculated summary id, but that should never happen because its only used in the context builder where the ids are definitely valid.